src/App.jsx

```diff
@@ -1,44 +1,34 @@
-import React from 'react';
+import React, { useState } from 'react';
 import 'bulma/css/bulma.css';
 import '@fortawesome/fontawesome-free/css/all.css';
 import './App.scss';
+import { Tabs } from './components/Tabs';
 
 export const tabs = [
   { id: 'tab-1', title: 'Tab 1', content: 'Some text 1' },
   { id: 'tab-2', title: 'Tab 2', content: 'Some text 2' },
   { id: 'tab-3', title: 'Tab 3', content: 'Some text 3' },
 ];
 
-export const App = () => (
-  <div className="section">
-    <h1 className="title">Selected tab is Tab 1</h1>
+export const App = () => {
+  const [activeTabId, setActiveTabId] = useState('invalid-tab-id');
```

Comment:
The initial state of `activeTabId` is set to `'invalid-tab-id'`, which does not correspond to any of the tab IDs in the `tabs` array. Consider setting it to a valid tab ID, such as `tabs[0].id`, to ensure that a valid tab is selected by default.